contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -11,14 +11,23 @@ toc::[]
 == Modularization backstory
 OpenShift docs are getting modularized, starting from OpenShift 4.0.
 All existing content will be completely replaced with content that is based on user stories and
-complies with the modularization guidelines. All future content must support a user story.
+complies with the modularization guidelines. All future content must both
+support a user story and be modular.
 
-== How do I contribute user stories?
-Familiarize yourself with the Customer Content Services (CCS)
-link:https://redhat-documentation.github.io/modular-docs/[modularization guide]. This is an important
-first step in making sure you are helping the docs team create content that is fully modularized.
+== How do I contribute modularized content?
+To contribute modularized content, you need to write a user story, create
+documentation modules to support the user story, and create an assembly for the
+story.
 
-== What is the branch setup for modularized docs?
+== What if I don't want to write in modules?
+If you don't want to write the modules yourself but have a content change,
+write a user story, provide details to support the story, and reach out to the
```

Comment:
Maybe a link to _How do I write a user story?_ for the reasons previously mentioned? So that folks know what kind of user story they need to create?